web_model_read_only/tests/test_models.py

```diff
@@ -0,0 +1,82 @@
+from odoo.exceptions import AccessError
+from odoo.tests.common import SavepointCase
+
+
+class TestIrModel(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestIrModel, cls).setUpClass()
+        cls.readonly_group = cls.env.ref("base.group_user")
+        cls.group_manager = cls.env.ref("base.group_erp_manager")
+        cls.ir_model = cls.env["ir.model"].create(
+            {"name": "x_test_model", "model": "x_test.model"}
+        )
+        User = cls.env["res.users"]
```

Comment:
Better change naming. For this situation and similar use: `res_users obj`. `obj` suffix used for the showing what variable contain empty model object.